docker/build.sh

```diff
@@ -2,21 +2,21 @@
 set -e
 
 # Edit these properties when moving to a new version.
-VERSION=3.3.0
+VERSION=4.1.0
 # SHA-256...
-SHA=b6b68dfcd0f5ba767a202bf35eb3eb964c63e679e8217dd514dac807e6cedee8
+SHA=5ad18a5a8702fcfe42d5180ee4ba33cf4728d5f68055b2d3755a86222afac462
 
 mkdir -p tmp
 if [ ! -f tmp/bnd.jar ]; then
 	echo Downloading bnd JAR version $VERSION
-	curl https://search.maven.org/remotecontent?filepath=biz/aQute/bnd/biz.aQute.bnd/$VERSION/biz.aQute.bnd-$VERSION.jar > tmp/bnd.jar
+	curl http://central.maven.org/maven2/biz/aQute/bnd/biz.aQute.bnd/$VERSION/biz.aQute.bnd-$VERSION.jar > tmp/bnd.jar
```

Comment:
I don't see why you made the change from search to central? And also removed https?